roles/servicetelemetry/templates/manifest_snmp_traps.j2

```diff
@@ -27,3 +27,16 @@ spec:
               value: "{{ snmp_trap_host }}"
             - name: SNMP_PORT
               value: "162"
+----
```

Comment:
This should probably be `---`, but I'd not bother putting this `Service` in the existing manifest, because if someone overrides the `Deployment` manifest, you'll lose the `Service` manifest. Break `Service` out to a separate `k8s` module call, and use the `definition` parameter so you don't need to use a template file at all.

Reply:
neither 3 nor 4 dashes '-' do the trick. I'll move to the separate service with the next pr here.

Reply:
> This should probably be `---`, but I'd not bother putting this `Service` in the existing manifest, because if someone overrides the `Deployment` manifest, you'll lose the `Service` manifest. Break `Service` out to a separate `k8s` module call, and use the `definition` parameter so you don't need to use a template file at all.

So, when going the `Service`route in a definition: how would I then make the namespace configurable?

Reply:
You'd just use the `{{ meta.namespace }}` parameter. If you've broken `Deployment` and `Service` into separate templates/`k8s` module calls, you won't need the hyphens anyways. It's simply meant as a multi-object separator.